gen/gen.go

```diff
@@ -128,18 +128,47 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 		Version: commonMetadata.Version,
 	}, j}
 
+	// Use two potential output files, one for success, and one for format.Source failure.
+	casesFileName := filepath.Join(dirExercise, "cases_test.go")
+	casesErrorFileName := filepath.Join(dirExercise, "cases_test.go-error")
```

Comment:
Perhaps a different name, such as `failed_cases_test.go`, or `error_cases_test.go`, just so it can still be picked up in an editor as a `go` file?